docs/releases/6.0.txt

```diff
@@ -207,7 +207,8 @@ Security
 Serialization
 ~~~~~~~~~~~~~
 
-* ...
+* The :ref:`JSON <serialization-formats-json>` serializer now writes a newline
+  at the end of the output, even when the ``indent`` option is ``True``.
```

Comment:
```suggestion
  at the end of the output, even without the ``indent`` option set.
```
I think it's the falsey case we are updating

I also think currently this would be a point in the backwards incompatible changes rather than the new features section

Reply:
Thanks. The embarassing part is that my AI autocomplete tried to tell me this at least three times and I thought I knew better 🤦🏻

Reply:
No worries 😁
I still think this should move to the backwards incompatible changes section rather than a new feature

Reply:
Yes, makes sense, moved to the misc section there.